### ISSUE TITLE
script error when in facebook

### ISSUE THREAD

**Issue by Murf1954** [mgziminsky/FacebookTrackingRemoval#29]
I get a yellow script error bar in the  menu bar saying  there is s script error do  i want to remove  it stop it or wait i like this extention  and want to continue using it but it  locks up Facebook when the script error shows up

**Comment by Murf1954** [comment]
please  if you could fix this it would be appreciated

**Comment by mgziminsky** [comment]
Fill in the "Bug Report" template and I will look into it. Otherwise I'm just going to close this issue since you didn't provide any of the necessary information.

**Comment by Murf1954** [comment]
what info do you need  i tried to  get a screen shot  for you but iyt goed to fadt for me to do it. I get a yellow   bar in address bar has a script error   it gives  3 options temporary disa ble it  stop it or  wait  what info do yo need and you did not have to be so mean about it   i am using your add on like the add on but  it causes trouble  so I guess if you have to be nasty to me  ill just not use it and  ell all of my followerss not to either

**Comment by Murf1954** [comment]
how do  i fill   the bug report

**Comment by Murf1954** [comment]
never mind  i will just not use it  as  i get a nasty remark from you  
 you did not  tell me what exactly you need

**Comment by Murf1954** [comment]
close it then guess you do not care  about  who uses this

**Comment by mgziminsky** [comment]
I'm not trying to be rude, but when you opened the issue, you completely disregarded all of the instructions it contained. All the information i need is explained when opening a new issue.

Also, to be clear, I made this extension for myself. I couldn't care less if you, or anyone else, are using it. I thought it worked well enough that I made it publicly available in case others might find it helpful as well. I don't ask for any compensation or make any sort of money from this, and am in no way obligated to provide any sort of support. I am happy to help when someone puts in the appropriate amount of effort to explain their problem, but I'm not going to waste my time trying to guess.
